Source/WebCore/platform/graphics/avfoundation/CDMFairPlayStreaming.cpp

```diff
@@ -248,8 +248,8 @@ RefPtr<SharedBuffer> CDMPrivateFairPlayStreaming::sanitizeMpts(const SharedBuffe
 
 const Vector<Ref<SharedBuffer>>& CDMPrivateFairPlayStreaming::mptsKeyIDs() {
     static NeverDestroyed<Vector<Ref<SharedBuffer>>> mptsKeyID = [] {
-        ASCIILiteral keyID = "TransportStreamIdentifier"_s;
-        Ref keyBuffer = SharedBuffer::create(keyID.characters(), keyID.length());
+        const auto keyID = "TransportStreamIdentifier"_span;
+        Ref keyBuffer = SharedBuffer::create(keyID);
         return Vector { 1, WTFMove(keyBuffer) };
```

Comment:
I would write it more like this:
```suggestion
        return Vector { 1, SharedBuffer::create("TransportStreamIdentifier"_span) };
```
And we might be able to put two sets of braces instead of the `1` to get an `initializer_list`?